en/developer_guide/api.rst

```diff
@@ -0,0 +1,115 @@
+API
+===
+
+API access
+----------
+
+The connection to the API is based on the OAuth2 protocol.
+The Oauth2 protocol provides to a client a secure access to API on behalf of a user.
+
+A client is represented by a key/secret pair.
+In order to use use the API, the client must request a token.
+
+The token is a string that refers to the client and optionally to the user. It is used to authenticate the client
+when he make a request to the API.
+
+
+Client
+------
+
+The API clients can be managed in the administration section of the back office.
+
+The creation of a client is done through the add form. In the form the key/secret pair is automatically generated by the server.
+In this form you can specify if the client is credential.
+
+Token
+-----
+
+Open Orchestra implements two way to get a token. Both of those strategies are described in the OAuth2 protocol
+
+Client credentials
+~~~~~~~~~~~~~~~~~~
+
+This strategy will provide you with a token linked only to an API client.
+
+To obtain it, send the request:
+
+.. code-block::
+
+    /oauth/access_token?grant_type=client_credentials
+
+with HTTP header:
+
+.. code-block::
+
+    Authorization: Basic ENCODED_PAIR
+
+Where ``ENCODED_PAIR`` is a base64 encoded version of the key/secret pair.
```

Comment:
it's capital that the string to encode is "key:secret" (obviously key and secret depend on the client), i would specifically write it.